tools/twix/src/main.rs

```diff
@@ -145,7 +146,8 @@ impl_selectable_panel!(
     EnumPlotPanel,
     RemotePanel,
     TextPanel,
-    VisionTunerPanel
+    VisionTunerPanel,
+    ImageColorSelectPanel
```

Comment:
```suggestion
    ImageColorSelectPanel,
```
#hendrikComma

Reply:
done (needed to fix the panel makro with ole)